parser.go

```diff
@@ -105,11 +111,36 @@ func (p *tagParser) parseFunction(f *ast.FuncDecl) {
 	}
 
 	p.tags = append(p.tags, tag)
+
+	if p.extraSymbols[Module] || p.extraSymbols[Receiver] {
```

Comment:
This whole block can be simplified if you merge the Module and Receiver fields. Also, use `p.extraSymbols.Includes` instead of directly accessing the `map`.